### PR TITLE
Documentation update for using Translation Generator in the new system texts editor

### PR DESCRIPTION
**Pull request by Yiyun333**

### CODE REVIEW

File: content/en/docs/refguide/modeling/mendix-ai-assistance/translation-generator.md [hunk 1]
```diff
@@ -33,7 +39,20 @@ Once translations are generated, you need to Click **Translate** to save the cha
 Currently, if you encounter an error after clicking **Generate Translations...**, try to sign out and sign in to Studio Pro again.
 {{% /alert %}}
 
+### Generating Translation for System Texts {#translate-system-text}
+
+In Studio Pro 10.14.0 and above, you can also use Translation Generator in the web version of the [system texts](/refguide/system-texts/) editor. 
+
+The web-based system texts editor was released in Studio Pro 10.14.0 as an experimental feature. You can enable it via **Preferences** > the **New Features** tab > the **System Texts Editor** section.
+
+After enabling the new editor and Translation Generator, make sure to restart Studio Pro. You will then see the **Generate translation** option from **App Explorer** > **App 'AppName'** > **System texts**.
```
Comment (Yiyun333) [inline content/en/docs/refguide/modeling/mendix-ai-assistance/translation-generator.md:48]: After I enabled the new editor, I had to restart Studio Pro to be able to see the translation oprion. I wrote it into the documentation. What do you think? @mauritselzinga-mx

File: content/en/docs/refguide/modeling/mendix-ai-assistance/translation-generator.md [hunk 2]
```diff
@@ -33,7 +39,20 @@ Once translations are generated, you need to Click **Translate** to save the cha
...
+
+{{< figure src="/attachments/refguide/modeling/mendix-ai-assistance/translation-generator/translate-system-text.png" max-width=80% >}}
+
+It generates translation for the system texts based on what the [current language](/refguide/translatable-texts/#current-language) is for app end-users.
```
Comment (Yiyun333) [inline content/en/docs/refguide/modeling/mendix-ai-assistance/translation-generator.md:52]: Is this desciption accurate? @mauritselzinga-mx

File: content/en/docs/refguide/modeling/mendix-ai-assistance/translation-generator.md [hunk 3]
```diff
@@ -33,7 +39,20 @@ Once translations are generated, you need to Click **Translate** to save the cha
 Currently, if you encounter an error after clicking **Generate Translations...**, try to sign out and sign in to Studio Pro again.
 {{% /alert %}}
 
+### Generating Translation for System Texts {#translate-system-text}
+
+In Studio Pro 10.14.0 and above, you can also use Translation Generator in the web version of the [system texts](/refguide/system-texts/) editor. 
+
+The web-based system texts editor was released in Studio Pro 10.14.0 as an experimental feature. You can enable it via **Preferences** > the **New Features** tab > the **System Texts Editor** section.
```
Comment (Yiyun333) [inline content/en/docs/refguide/modeling/mendix-ai-assistance/translation-generator.md:46]: Do we need to add a bit more information on what web version of the system texts editor actually does or mean? @mauritselzinga-mx

Comment (ghost) [reply content/en/docs/refguide/modeling/mendix-ai-assistance/translation-generator.md:46]: No, it's a drop in replacement of the technology.

File: content/en/docs/refguide/modeling/mendix-ai-assistance/translation-generator.md (hunk 1 above)
Comment (ghost) [reply content/en/docs/refguide/modeling/mendix-ai-assistance/translation-generator.md:48]: If that restart is required, then great to add it!

File: content/en/docs/refguide/modeling/mendix-ai-assistance/translation-generator.md (hunk 2 above)
Comment (ghost) [reply content/en/docs/refguide/modeling/mendix-ai-assistance/translation-generator.md:52]: yes, that is correct

**Comment by Yiyun333** [comment]
This PR is good to go for the release.

**Pull request merged**